versioned_docs/version-1.3/configuration/plugins/example-configurations.md

```diff
@@ -1002,3 +1002,41 @@ Once installed and synced, add your WakaTime API Key via `:WakaTimeApiKey` comma
 },
 
 ```
+
+### [flutter-tools](https://github.com/tpope/vim-rails)
```

Comment:
You forgot something here 🥹

Reply:
Whoops! It looks like I made a mistake. That's a link to vim-rails. I'll make another commit. Sorry about that.